apps/mobile/src/features/send-refactor/utils.ts

```diff
@@ -0,0 +1,125 @@
+import { type BtcFormSchema } from '@/features/send-refactor/forms/btc/btc-form-schema';
```

Comment:
This `utils` file seems to be a dumping ground of multi chain functions. 

Chain specific code should live in the respective package - `leather.io/stacks` or `leather.io/bitcoin` .

`validateAddressCompliance` shouldn't mix checks on stacks and bitcoin address

Reply:
I agree. some are there mostly due to my lack of knowledge on how to promote them into library-level helpers, e.g. `btcFormValuesToPsbtHex`is form-specific, it was migrated from an inline logic in huge hook from the previous version, but `validateAddressCompliance` is poorly implemented, it should accept a chain-specific address validator as a callback (the way it's done in the extension) instead of baking it in.

Reply:
Sure: [LEA-2384: Refactor & Test Send flow helpers](https://linear.app/leather-io/issue/LEA-2384/refactor-and-test-send-flow-helpers)